src/types/database.ts

```diff
@@ -1,3 +1,5 @@
+// cSpell:disable
```

Comment:
Since this is auto-generated I think you'd be better setting the ignore of this file through the configuration.